### PR TITLE
chore(HMS-3968): update http scripts

### PR DESCRIPTION
**Pull request by avisiedo**
I have not found at a way to select between the different XRHID
but it is provided for developing propose one and how to quickly
generate a new one for developing propose.

Depends on: https://github.com/podengo-project/idmsvc-backend/pull/244

> This can be merged independent, but it does not make sense to merge this change without the others, so adding the label `deps_on_pr`.

Additional changes detected during the review:

- Fix PATCH /domains/:domain_id operation was not including
  `X-Rh-Insights-Request-Id` header.
- Remove hardcoded value for `X-Rh-Identity` header for
  `PUT /domains/:domain_id`
- Fix path to check the access at rbac.http: `/api/rbac/v1/access?application=idmsvc`

### CODE REVIEW

File: scripts/http/public.http [hunk 1]
```diff
@@ -6,6 +6,13 @@
...
+# Generated by:
+# ./tools/bin/xrhidgen \
+#   -org-id 12345 -account-number 11111 -employe-account-number 22222 \
+#   service-account \
+#   -username jdoe -client-id "91a229a3-ea92-4ec9-a926-8885d67e6a56"
+@x-rh-identity-service-account = eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6IjExMTExIiwiZW1wbG95ZWVfYWNjb3VudF9udW1iZXIiOiIyMjIyMiIsIm9yZ19pZCI6IjEyMzQ1IiwiaW50ZXJuYWwiOnsib3JnX2lkIjoiMTIzNDUifSwidXNlciI6eyJ1c2VybmFtZSI6Impkb2UiLCJlbWFpbCI6Impkb2VAZXhhbXBsZS5jb20iLCJmaXJzdF9uYW1lIjoiSm9obiIsImxhc3RfbmFtZSI6IkRvZSIsImlzX2FjdGl2ZSI6dHJ1ZSwiaXNfb3JnX2FkbWluIjp0cnVlLCJpc19pbnRlcm5hbCI6dHJ1ZSwibG9jYWxlIjoiZW4iLCJ1c2VyX2lkIjoiMTk4NzM0OCJ9LCJzeXN0ZW0iOnt9LCJhc3NvY2lhdGUiOnsiUm9sZSI6bnVsbCwiZW1haWwiOiIiLCJnaXZlbk5hbWUiOiIiLCJyaGF0VVVJRCI6IiIsInN1cm5hbWUiOiIifSwieDUwOSI6eyJzdWJqZWN0X2RuIjoiIiwiaXNzdWVyX2RuIjoiIn0sInR5cGUiOiJVc2VyIiwiYXV0aF90eXBlIjoiIn19Cg==
```
Comment (pvoborni) [inline scripts/http/public.http:14]: This identity is not service account. It's a copy&paste of the user account. Same issue in `rbac.http`.

```
 echo "eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6IjExMTExIiwiZW1wbG95ZWVfYWNjb3VudF9udW1iZXIiOiIyMjIyMiIsIm9yZ19pZCI6IjEyMzQ1IiwiaW50ZXJuYWwiOnsib3JnX2lkIjoiMTIzNDUifSwidXNlciI6eyJ1c2VybmFtZSI6Impkb2UiLCJlbWFpbCI6Impkb2VAZXhhbXBsZS5jb20iLCJmaXJzdF9uYW1lIjoiSm9obiIsImxhc3RfbmFtZSI6IkRvZSIsImlzX2FjdGl2ZSI6dHJ1ZSwiaXNfb3JnX2FkbWluIjp0cnVlLCJpc19pbnRlcm5hbCI6dHJ1ZSwibG9jYWxlIjoiZW4iLCJ1c2VyX2lkIjoiMTk4NzM0OCJ9LCJzeXN0ZW0iOnt9LCJhc3NvY2lhdGUiOnsiUm9sZSI6bnVsbCwiZW1haWwiOiIiLCJnaXZlbk5hbWUiOiIiLCJyaGF0VVVJRCI6IiIsInN1cm5hbWUiOiIifSwieDUwOSI6eyJzdWJqZWN0X2RuIjoiIiwiaXNzdWVyX2RuIjoiIn0sInR5cGUiOiJVc2VyIiwiYXV0aF90eXBlIjoiIn19Cg==" | base64 -d | jq
{
  "identity": {
    "account_number": "11111",
    "employee_account_number": "22222",
    "org_id": "12345",
    "internal": {
      "org_id": "12345"
    },
    "user": {
      "username": "jdoe",
      "email": "jdoe@example.com",
      "first_name": "John",
      "last_name": "Doe",
      "is_active": true,
      "is_org_admin": true,
      "is_internal": true,
      "locale": "en",
      "user_id": "1987348"
    },
    "system": {},
    "associate": {
      "Role": null,
      "email": "",
      "givenName": "",
      "rhatUUID": "",
      "surname": ""
    },
    "x509": {
      "subject_dn": "",
      "issuer_dn": ""
    },
    "type": "User",
    "auth_type": ""
  }
}
```

**Review by pvoborni — CHANGES_REQUESTED** [review]
There is one major issue, otherwise LGTM.

File: scripts/http/public.http (hunk 1 above)
Comment (avisiedo) [reply scripts/http/public.http:14]: :+1: good catch!!!  thanks for this!

Comment (pvoborni) [reply scripts/http/public.http]: The issue seems to be still present in the `rbac.http` (as also mentioned in prev. comment).

Comment (avisiedo) [reply scripts/http/public.http]: thanks for the heads up! I am on it!

**Pull request merged**